Components/Badge/badge.css

```diff
@@ -0,0 +1,116 @@
+.badge-round-avatar {
+    border-radius: 9999px;
+    width: 25px;
```

Comment:
try using responsive units instead of absolute values

Reply:
Done. Thanks for pointing it out :)